src/main/java/com/conveyal/taui/models/OpportunityDataset.java

```diff
@@ -0,0 +1,34 @@
+package com.conveyal.taui.models;
+
```

Comment:
We should clarify our terminology: dataset seems like a collective plural, but this class represents a single spatial layer.

Suggested javadoc:
Layer for a single attribute of an imported shapefile or seamless-census extract